test/testvm.py

```diff
@@ -752,45 +750,41 @@ def _cleanup(self):
             print >> sys.stderr, "WARNING: Cleanup failed:", str(value)
 
     def kill(self):
+        # attempt graceful shutdown first
```

Comment:
I had to add this because the regular kill leaves zombie qemu sessions on my system...